### PR TITLE
12 feature creating homepage

### PR DESCRIPTION
**Pull request by maxswaine**
Created homepage using other features including Button, NavBar and Header

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -79,6 +79,11 @@ const fakeStyle = {
 - We currently have 5 mock events that have been taken from the Figma page with the ability to add further events as and when needed.
 - we have added 2 events with a data that's in past so they can be used to show in the "Historic" section.
 
+## Homepage
+
+Created a homepage that replicated
```
Comment (Charlie-robin) [inline README.md:84]: What did it replicate??

File: src/pages/Home/Home.scss [hunk 2]
```diff
@@ -0,0 +1,32 @@
+@use "../../styles/variables" as *;
+
+.homepage {
+  margin: 2.5rem;
+}
+
+button {
```
Comment (Charlie-robin) [inline src/pages/Home/Home.scss:7]: Remove this the `<Button>` has a prop to allow you to do this kind of CSS tweaks.

File: src/pages/Home/Home.tsx [hunk 3]
```diff
@@ -1,20 +1,89 @@
...
+        <br></br>
+        <br></br>
+        <p className="homepage__content">
+          Learn more about the history of pottery and how ceramics can be used
+          as a building material for small clay bricks. Using the tools you
+          learned from the workshop you will make a few pieces.
         </p>
-      </Layout>
-    </div>
+        <Button label="ABOUT" onClick={handleViewAbout} />
```
Comment (Charlie-robin) [inline src/pages/Home/Home.tsx:83]: Use the style prop on the button should look something like this.

```tsx
<Button style={{marginBottom: "2rem", marginTop: "2rem"}} label="ABOUT" onClick={handleViewAbout}/>
```

File: src/pages/Home/Home.scss [hunk 4]
```diff
@@ -0,0 +1,32 @@
+@use "../../styles/variables" as *;
+
+.homepage {
```
Comment (Charlie-robin) [inline src/pages/Home/Home.scss:3]: Can you nest your styles and use the `&` to match the other components in the project, please?

**Review by Charlie-robin — CHANGES_REQUESTED** [review]
Can you resolve the merge conflicts and have a look at the comments below please.

**Review by olirob93 — APPROVED** [review]
looks good to me

**Review by Charlie-robin — APPROVED** [review]
Looking good!

**Pull request merged**